packages/devtools_app/lib/src/screens/debugger/controls.dart

```diff
@@ -138,6 +140,50 @@ class _DebuggingControlsState extends State<DebuggingControls>
   }
 }
 
+class CodeCoverageToggle extends StatelessWidget {
+  const CodeCoverageToggle({
+    super.key,
+    required this.controller,
+  });
+
+  final DebuggerController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return RoundedOutlinedBorder(
```

Comment:
can we use `DevToolsToggleButtonGroup` here so that we get the default devtools styling for this toggle button? example here: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/service/service_extension_widgets.dart#L676

Reply:
Or you can change ToggleButton to use the same styling that `DevToolsToggleButtonGroup` does. This may be better since all the uses of ToggleButton would also get this change.

Reply:
Is this what you were thinking?